src/Bands/Bands.js

```diff
@@ -2,17 +2,58 @@ import React, { Component } from 'react';
 import Band from '../Band/Band.js';
 
 export default class Bands extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
+    this.state = {
+      active: this.props.genreBands.filter(band => {
+        return band.active === true;
+      }),
+      inactive: this.props.genreBands.filter(band => {
+        return band.active === false;
+      })
+    }
   }
 
   render() {
```

Comment:
This `render` is getting pretty long with all the different variations that can be returned based on props - making it difficult to read. Might be a good idea to look into some ways of handling this, like [sub-rendering](https://www.barrymichaeldoyle.com/sub-rendering/) .